product_standard_price_restriction/security/product_security.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="group_product_manager" model="res.groups">
```

Comment:
@rinaldifirdaus Do you think we should add this group this time, according to the requirement discussed with the client?